XRTK-Core/Packages/com.xrtk.core/Runtime/Definitions/Controllers/Hands/RecordedHandJoint.cs

```diff
@@ -20,43 +20,24 @@ public struct RecordedHandJoint
         /// <param name="pose"></param>
         public RecordedHandJoint(TrackedHandJoint joint, MixedRealityPose pose)
         {
-            this.joint = JointNames[(int)joint];
+            this.joint = joint;
             this.pose = pose;
         }
 
-        private static readonly string[] JointNames = Enum.GetNames(typeof(TrackedHandJoint));
+        [SerializeField]
+        private TrackedHandJoint joint;
 
         /// <summary>
-        /// Name of the joint recorded. Rather use <see cref="JointIndex"/> below for convenience.
+        /// Joint this pose represents.
         /// </summary>
-        public string joint;
+        public TrackedHandJoint Joint => joint;
 
-        /// <summary>
-        /// The recorded pose.
-        /// </summary>
-        public MixedRealityPose pose;
+        [SerializeField]
+        private MixedRealityPose pose;
 
         /// <summary>
-        /// Gets the <see cref="TrackedHandJoint"/> this record represents.
+        /// The recorded pose.
         /// </summary>
-        public TrackedHandJoint JointIndex
```

Comment:
This seemed a little convoluted and unnecessary so I just replaced it with the `TrackedHandJoint` enum directly.
Enums are pretty good about turning back into their named value, and it didn't seem like we were using the string values but to convert back and forth, so I figured it could go.

Reply:
True, thanks.